src/pages/AboutUs/Details.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import pic from './pic.png';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+
+function Details() {
+  return (
+    <MDBContainer className="bg-gray-100">
+      <MDBRow>
+        <MDBCol lg="5">
+          <img src={pic} alt="artwork of a girl" />
+        </MDBCol>
+        <MDBCol>
+          <div className="mt-40">
+            <h3 className="font-weight-bold  p-0  text-3xl ml-1">
+              <strong>About us :</strong>
```

Comment:
This text as well as other texts should be module, I mean you should have an object and you put this text inside it and 
 then exported inside this tag

```
const aboutPageEnglishTransslation = {
       headerTitle: "About us"
}
```

and then you can use it like this 
```
<strong>{aboutPageEnglishTransslation.headertitle} :</strong>
````

so it will be easy to put the translation using i18next 

please refactor all your code like that and also remember that we have translation for three languages